src/vllm_tgis_adapter/__main__.py

```diff
@@ -45,18 +50,30 @@
 
     from vllm.config import ModelConfig
 
+
+try:
```

Comment:
Tokenization endpoints were introduced after 0.5.2 was released, leaving this here to prepare for next releases

Reply:
@dtrifiro the tokenization & detokenization endpoints were introduced in 0.5.1?